examples/19_DFSM/modeling_options_RM1.yaml

```diff
@@ -0,0 +1,192 @@
+General:
+    verbosity: False  # When set to True, the code prints to screen many infos
+    openfast_configuration:
+        OF_run_fst: RM1
+        # OF_run_dir: outputs/05_low_shear_hi_TI
+        save_timeseries: True
+        save_iterations: True
+        use_exe: True
+        allow_fails: True
+        fail_value: 9999
+        FAST_exe: /home/athulsun/anaconda3/envs/openfast_env/bin/openfast
```

Comment:
Can you remove local copies of these for the inputs you plan on adding to the repo?